### PR TITLE
[IMP] gauge: add choice of operator on inflection point

### PR DESCRIPTION
**Pull request by hokolomopo**
## Description:

This commit allows to chose from the operators "<" and "<=" when
defining the inflection point of a gauge chart.

Task: [4170179](https://www.odoo.com/web#id=4170179&action=333&active_id=2328&model=project.task&view_type=form&cids=1&menu_id=4720)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_t("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

File: src/migrations/data.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import { migrationStepRegistry } from "./migration_steps";
  * a breaking change is made in the way the state is handled, and an upgrade
  * function should be defined
  */
-export const CURRENT_VERSION = 20;
+export const CURRENT_VERSION = 21;
```
Comment (LucasLefevre) [inline src/migrations/data.ts:22]: we are now at version 21 already, it should be 22 here :sweat_smile: :rocket:

Comment (hokolomopo) [reply src/migrations/data.ts]: Rebased :)

File: src/components/side_panel/chart/gauge_chart_panel/gauge_chart_design_panel.xml [hunk 2]
```diff
@@ -94,7 +95,17 @@
...
-      <td>When value is below</td>
+      <td>When value is</td>
+      <td class="pe-2">
+        <select
+          class="o-input"
+          name="operatorType"
+          t-model="inflectionPoint.operator"
+          t-on-change="() => this.updateSectionRule(state.sectionRule)">
+          <option value="&lt;">&lt;</option>
+          <option value="&lt;=">&lt;=</option>
```
Comment (LucasLefevre) [inline src/components/side_panel/chart/gauge_chart_panel/gauge_chart_design_panel.xml:106]:
```patch
diff --git a/src/components/side_panel/chart/gauge_chart_panel/gauge_chart_design_panel.xml b/src/components/side_panel/chart/gauge_chart_panel/gauge_chart_design_panel.xml
index c52a6bd11..da5b088b2 100644
--- a/src/components/side_panel/chart/gauge_chart_panel/gauge_chart_design_panel.xml
+++ b/src/components/side_panel/chart/gauge_chart_panel/gauge_chart_design_panel.xml
@@ -97,13 +97,16 @@
       </td>
       <td>When value is</td>
       <td class="pe-2">
+        <t t-set="below">below</t>
+        <t t-set="belowOrEqualTo">below or equal to</t>
         <select
           class="o-input"
+          t-att-title="inflectionPoint.operator === '&lt;=' ? belowOrEqualTo : below"
           name="operatorType"
           t-model="inflectionPoint.operator"
           t-on-change="() => this.updateSectionRule(state.sectionRule)">
-          <option value="&lt;">&lt;</option>
-          <option value="&lt;=">&lt;=</option>
+          <option title="below" value="&lt;">&lt;</option>
+          <option title="below or equal to" value="&lt;=">&lt;=</option>
         </select>
       </td>
       <td class="pe-2">

```

**Review by LucasLefevre — APPROVED** [review]
robodoo rebase-ff r+

**Pull request closed without merging**